src/bases/dictionary/gram.jl

```diff
@@ -51,10 +51,10 @@ innerproduct2(Φ1, i, Φ2::Dictionary, j, measure; options...) =
 
 
 # Make a quadrature strategy using user-supplied atol and rtol if they were given
-function quadstrategy(::Type{T}; atol = 0, rtol = -1, options...) where {T}
+function quadstrategy(::Type{T}; atol = 0, rtol = -1, maxeval=1e4 , options...) where {T}
```

Comment:
This works, but a small downside is that the constant 1e4 here is somewhat arbitrary. I've just updated DomainIntegrals.jl to define default_atol(), default_rtol() and default_maxevals() functions, so that we could use it here.

So this function could become:

function quadstrategy(::Type{T}; atol = DomainIntegrals.default_atol(T), rtol = DomainIntegrals.default_rtol(atol), maxevals=DomainIntegrals.default_maxevals() , options...) where {T}
    QuadAdaptive{T}(atol, rtol, maxevals)
end

I've used maxevals in DomainIntegrals (not maxeval), just like in HCubature.jl

Reply:
Excellent idea, the above is updated in the branch, should automatically merge through the pull request.